liquidation-monitor/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "forta-agent-starter",
+  "version": "0.0.1",
+  "description": "Forta Agent Javascript starter project",
+  "chainIds": [
+    1
+  ],
+  "scripts": {
+    "start": "npm run start:dev",
+    "start:dev": "nodemon --watch src --watch forta.config.json -e js,json --exec \"forta-agent run\"",
+    "start:prod": "forta-agent run --prod",
+    "tx": "forta-agent run --tx",
+    "block": "forta-agent run --block",
+    "range": "forta-agent run --range",
+    "file": "forta-agent run --file",
+    "publish": "forta-agent publish",
+    "push": "forta-agent push",
+    "disable": "forta-agent disable",
+    "enable": "forta-agent enable",
+    "keyfile": "forta-agent keyfile",
+    "test": "jest"
+  },
+  "dependencies": {
+    "bignumber.js": "^9.0.2",
+    "forta-agent": "^0.1.3",
+    "node-fetch-commonjs": "^3.1.1"
```

Comment:
Were we still hoping to use `axios` or stick with `fetch`?

Reply:
Fetch works well, I will leave it as is.